### PR TITLE
Configure vale more

### PR DESCRIPTION
**Pull request by programmiri**
# Description

This PR adds:
- proselint Vale style for suggestions
- small style rules (like [whitespace-punctation.yml](https://github.com/Aiven-Open/klaw-docs/pull/97/files#diff-885c2d29dc315eb0bafc6c08f2a044561d4b4aa406715e24e46d2eee48006b4c) and [extraneous-apostrophes.yml](https://github.com/Aiven-Open/klaw-docs/pull/97/files#diff-f99c7cf81649d5edfb59df7f51d4cb2212f7928a4faba01d0862d5c372e15ce1)) to avoid errors in the future.
- a Klaw Vale style for branding and branding-warning-temp
--> `branding` is what marks words as error. `branding-warning-temp` is a temporary file with the goal to give outside contributors options for small doable PRs by moving one branding-rule to the error file and fix the issues. 
- adds [capitalization-heading.yml](https://github.com/Aiven-Open/klaw-docs/pull/97/files#diff-b63d0b2af103596c9bce325fe4dd3a2cddf95c71222d7dc7482c110356f8d25d) with level warning, based on Aiven's devportal styles (plus our custom ones)
- adds [first-kafka-registered.yml](https://github.com/Aiven-Open/klaw-docs/pull/97/files#diff-5fe57b4e021133fd8bedc8233598c33e0f883e6a5bf06504e7cc36f78fba9a8f) with level warning (there is a follow up issue for hacktoberfest)
- adds [first-kafka-registered.yml](https://github.com/Aiven-Open/klaw-docs/pull/97/files#diff-5fe57b4e021133fd8bedc8233598c33e0f883e6a5bf06504e7cc36f78fba9a8f) for follow up issue on hacktoberfest

### CODE REVIEW

File: .github/vale/styles/proselint/Airlinese.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+extends: existence
+message: "'%s' is airlinese."
```
Comment (mathieu-anderson) [inline .github/vale/styles/proselint/Airlinese.yml:2]: What does "airlinese" mean? :o

File: .github/vale/styles/proselint/AnimalLabels.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+extends: substitution
+message: "Consider using '%s' instead of '%s'."
```
Comment (mathieu-anderson) [inline .github/vale/styles/proselint/AnimalLabels.yml:2]: Hahaha wow that is very specific ^^

File: .github/vale/styles/proselint/Archaisms.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  - haply
+  - howbeit
+  - illumine
+  - in sooth
+  - maugre
+  - meseems
+  - methinks
+  - nigh
+  - peradventure
+  - perchance
```
Comment (mathieu-anderson) [inline .github/vale/styles/proselint/Archaisms.yml:34]: Perchance to dream...

File: .github/vale/styles/proselint/Cliches.yml [hunk 4]
```diff
@@ -0,0 +1,782 @@
+extends: existence
+message: "'%s' is a cliche."
```
Comment (mathieu-anderson) [inline .github/vale/styles/proselint/Cliches.yml:2]: This one seems pretty brutal, and probably more suited to creative writing than to documentation? Clichés are good to communicate commonplace ideas, which sometimes help to explain software :o

**Review by mathieu-anderson — APPROVED** [review]
Seems good to me, only the "cliché" rule seems a little overbearing to me :o

File: .github/vale/styles/proselint/Airlinese.yml (hunk 1 above)
Comment (programmiri) [reply .github/vale/styles/proselint/Airlinese.yml:2]: I think it means something like "airline language"

The rules that we add from external (like "google" and "proselint") are automatically and should not changed (they update on every `vale sync`), that's why there are some in that we may not need and are only suggestions or even turned off. 

The rules basically come as a "set" :D and we choose which from them we want to use in `vale.ini`.

File: .github/vale/styles/proselint/Cliches.yml (hunk 4 above)
Comment (programmiri) [reply .github/vale/styles/proselint/Cliches.yml:2]: Yes :D Same as above, the rules come in a "set", so they are all added when we run `vale sync`. proselint is only for suggestions right now. 

I hope in the long run, we can establish a few rules as "hard" ones and the rest can be used to get a bit input, same way we can use `alex` over in coral, but it's not an established rule.

**Pull request merged**